ruby/example_code/dynamodb/README.md

```diff
@@ -1,4 +1,4 @@
-# DynamoDB code examples for the AWS SDK for Ruby
```

Comment:
Weren't you going to remove this file from the PR?

Reply:
Yep! Just didn't get around to this yesterday. Done!